src/MetaModels/Events/DefaultPropertyConditionCreator.php

```diff
@@ -70,6 +72,22 @@ public function handle(CreatePropertyConditionEvent $event)
                     $meta['value']
                 ));
                 break;
+            case 'conditionpropertymultiplevalueis':
```

Comment:
After reading again, we should rather change the handling in https://github.com/MetaModels/core/blob/tng/src/MetaModels/BackendIntegration/InputScreen/InputScreen.php#L272 rather than changing here as this is already an event handler but I think changes over there will imply changes here as well.
Sorry for the confusion.

Do you have any good idea of how to improve the flexibility?